payload-types.ts

```diff
@@ -188,7 +189,7 @@ export interface SportsTeam {
  */
 export interface StudentGuide {
   id: number;
-  language: 'Português' | 'Inglês';
+  title: 'Português' | 'Inglês';
```

Comment:
is this intentionally?

Reply:
yes, because we are using the same component (PDF) to show documents and StudentGuides pdf iframes. The consistency in the name makes it easier, but if there is a better way, we are free.